app/ai_backend/test_genesis_consciousness_matrix.py

```
@@ -10,7 +10,7 @@
 import pytest
 import tempfile
 import unittest
-from datetime import datetime, timedelta
+from unittest.mock import patch, MagicMock, mock_open
 from typing import Dict, List, Any
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)